python/recognize_rest.py

```diff
@@ -1,31 +1,34 @@
 #! /usr/bin/env python3
-import requests
+import asyncio
+import httpx
 from audio import audio_open_read
 from auth import authorization_metadata
 from common import build_recognition_request, print_recognition_response, BaseRecognitionParser
 
 from tinkoff.cloud.stt.v1 import stt_pb2
 
 
-def main():
+async def main():
     args = BaseRecognitionParser().parse_args()
 
     if args.encoding == stt_pb2.RAW_OPUS:
         raise ValueError("RAW_OPUS encoding is not supported by this script")
     with audio_open_read(args.audio_file, args.encoding, args.rate, args.num_channels, args.chunk_size,
                          args.pyaudio_max_seconds) as reader:
-        metadata = authorization_metadata(args.api_key, args.secret_key, "tinkoff.cloud.stt", type=dict)
-        request = build_recognition_request(args, reader, type="json")
-        response = requests.post("http{}://{}/v1/stt:recognize".format("s" if args.endpoint.endswith("443") else "",
+
+        async with httpx.AsyncClient(http2=True) as client:
```

Comment:
i would do it like another example

Reply:
first time I also thought to accomplish it as a separate example, but I. Lapshenkov said is not to show target users examples which working bases on http1 (the requests module in python doesn't support http2), because we are going to have a dedicated proxy for http1

by the way, if you will try to synthesize really long text using rest and http1, our balancer will return an internal error, but if you switch to http2, everything will be fine